css/style.css

```
@@ -321,3 +323,104 @@ color: var(--basic-white);
   font-size: 14px;
   line-height: 18px;
 }
+
+.filter-button {
+  font-family: PT Sans;
+  font-size: 13px;
+  line-height: 18px;
+  list-style: none;
+}
+
+.filter-button a {
+  color: var(--old-price);
+}
+
+.pagination-active {
+  font-family: PT Sans;
+  font-size: 13px;
+  line-height: 18px;
+  color: var(--basic-white);
+  background-color: var(--basic-red);
+}
+
+.pagination-none {
+  font-family: PT Sans;
+  font-size: 13px;
+  line-height: 18px;
+  color: var(--basic-black)
+}
+
+.filter-button:hover a,
+.filter-button:focus a,
+.filter-button:active a {
+  color: var(--basic-red);
+}
+
+.filter-show {
+  font-family: PT Sans;
+  font-size: 13px;
+  line-height: 18px;
+  background-color: var(--basic-white);
+  border: 1px solid #E5E5E5;
+}
+
+/*Catalog main*/
+.main-catalog {
+  background-color: var(--basic-white);
+}
+
+.form {
+  border: 0 none;
```

```suggestion
  border: 0 none;
```
Так быть не может. Чтобы скрыть границу - нужно использовать что-то одно. Рекомендую `none`